src/cloudinventario_amazon_aws/collector.py

```diff
@@ -56,33 +56,54 @@ def _fetch(self, collect):
   def _get_instance_type(self, itype):
     if itype not in self.instance_types:
       types = self.client.describe_instance_types(InstanceTypes = [ itype ])
+      storage = self._get_storage_info()
       for rec in types['InstanceTypes']:
         name = rec['InstanceType']
         data = {
           "cpu": rec['VCpuInfo']['DefaultVCpus'],
           "memory": rec['MemoryInfo']['SizeInMiB'],
-          "storage": None,
-          "storages": []
+          "storage": storage['TotalSize'],
+          "storages": storage['Storages']
         }
         if rec.get("InstanceStorageInfo"):
           data["storage"] = rec['InstanceStorageInfo']['TotalSizeInGB'] * 1024,
           for disk in rec['InstanceStorageInfo']['Disks']:
             data['storages'].append({
               "id": None,
               "name": None,
-              "capacity": disk['SizeInGB'] * 1024,
+              "capacity": disk['SizeInGB'] * 1024,  # in MB
               "free": None,
               "type": disk['Type'],
-              "ssd": (disk['Type'] == 'ssd') or 0,
+              "ssd": (disk['Type'] == 'ssd') or 0
             })
-          data['detail'] = rec
+          data['details'] = rec
         self.instance_types[name] = data
 
     if itype not in self.instance_types:
       raise Exception("Instance type '{}' not found".format(itype))
 
     return self.instance_types[itype]
 
+  def _get_storage_info(self):
+    storage = {
+      "TotalSize": 0,
+      "Storages": []
+    }
+    vinfo = self.client.describe_volumes()
```

Comment:
here, fetching **all** volumes, and this works only if there is one instance with 1 or more volumes, with more instances they get mixed